src/bb-modules/Servicedomain/Api/Guest.php

```diff
@@ -134,7 +134,7 @@ public function can_be_transferred($data)
         if (!$tld instanceof \Model_Tld) {
             throw new \Box_Exception('TLD is not active.');
         }
-        if (!$this->getService()->canBeTransfered($tld, $data['sld'])) {
+        if (!$this->getService()->canBetransferred($tld, $data['sld'])) {
```

Comment:
?

Reply:
What Is the Correct Spelling: Transferred or Transfered? “Transferred” is the only correct way of spelling this verb. “Transfered” is not a word and can't be used in any situation whatsoever.

Reply:
I was talking about the capitalization. I think it should be `canBeTransferred` if "transferred" is correct.